src/main/java/duke/command/Command.java

```diff
@@ -1,22 +1,20 @@
 package duke.command;
 
-import duke.command.ingredientCommand.ExitCommand;
+import duke.dish.DishList;
 import duke.exception.DukeException;
-import duke.list.GenericList;
-import duke.storage.Storage;
+import duke.ingredient.IngredientsList;
+import duke.order.OrderList;
+import duke.storage.FridgeStorage;
+import duke.storage.OrderStorage;
 import duke.ui.Ui;
 
-import java.io.IOException;
+public abstract class Command {
```

Comment:
I think we should keep the execute method abstract, so forcing all the commands to override it.